boards/arm/pinnacle_100_dvk/pinnacle_100_dvk.yaml

```diff
@@ -1,19 +1,23 @@
-identifier: adafruit_feather_nrf52840
-name: Adafruit Feather nRF52840 Express
```

Comment:
Did you mean to move this file? That will leave the Adafruit board without a yaml file

Reply:
Rebase error